src/main/java/randoop/contract/CompareToEquals.java

```diff
@@ -10,10 +10,13 @@
  * <pre>(x0.compareTo(x1) == 0) == x0.equals(x1)</pre>
  */
 public class CompareToEquals extends ObjectContract {
+  /** The singleton instance of this class. */
   private static final CompareToEquals instance = new CompareToEquals();
 
-  private CompareToEquals() {}
+  /** Singleton pattern. */
+  protected CompareToEquals() {}
```

Comment:
Why is this protected?  What other method needs to call it?  Shouldn't all clients call `getInstance`?

Reply:
CompareToEqualsWithImpurity (subclass of CompareToEquals) calls it to initialize itself

Reply:
That makes sense.  Please add a comment to that effect.  However, I later suggested removing CompareToEqualsWithImpurity, in which case this could be made private again.